playwright/test/search-works.test.ts

```diff
@@ -43,7 +42,6 @@ const expectSearchParam = (
 };
 
 const openDropdown = async (label: string, page: Page) => {
-  console.info('openDropdown', label);
```

Comment:
let's leave these in 🙏  (useful for debugging)

Reply:
Restored!

Reply:
I still see them as removed... I don't know if github is still lagging?

Reply:
oops 🙈 

no, in fact what actually happened is that I split those line changes into a separate commit that I could delete … but did not actually delete that commit

how about now?

Reply:
Yesss!